BottomBar.Droid/Renderers/BottomBarPageRenderer.cs

```diff
@@ -227,7 +231,8 @@ void UpdateBarTextColor ()
 				return;
 			}
 
-			// haven't found yet how to set text color for tab items on_bottomBar, doesn't seem to have a direct way
+            _bottomBar.SetActiveTabColor(Element.BarTextColor.ToAndroid());

```

Comment:
not sure, 
maybe create a new branch and just push PR from it with only code which is changed?
not sure how will we handle the situation when a new file is added to the .csproj....
another idea would be to just go ahead with the PR and stick to Visual Studio from now on, but it would require some reformatting of whole code...